packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -939,7 +939,12 @@ export default ({
         Number(cryptoAmt),
         account.coin
       )
-      payment = yield payment.amount(paymentAmount)
+      if (payment.coin === 'BTC' || payment.coin === 'BCH') {
```

Comment:
Is this a good idea?

Reply:
@plondon did you have a look at this?  any other ideas?

Reply:
I changed this bit of code here to include all coins, (removing the BTC and BCH check), which fixed an issue for ETH but created another issue for XLM. When trying to send, I would see an error about reserve currency being greater than balance. 

https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/86fbecd18239fdc28b7598c11f14ab5317da19aa/packages/blockchain-wallet-v4-frontend/src/data/components/swap/sagas.ts#L132